src/components/review/ReviewsList.js

```diff
@@ -4,17 +4,17 @@ import WriteReview from './WriteReview';
 import { Alert } from '@mui/material';
 import { useAuth } from '@akosasante/react-auth-context';
 import DeleteReview from './DeleteReview';
+import UpdateReview from './EditReview';
 
 const AlbumReviews = ({ albumId }) => {
   const [reviews, setReviews] = useState([]);
   const [loading, setLoading] = useState(true);
   const [error, setError] = useState(null);
-
-  const { user } = useAuth(); //use user.user.<whatever field we want> to access it properly
+  const [updateClicked, setUpdateClicked] = useState(false);
```

Comment:
Just a nitpick; I would probably call this something like `isEditing` instead of `updateClicked` but that's just a wording preference